helm-chart/renku-graph/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: '1.0'
 description: A Helm chart for renku graph services
 name: renku-graph
-version: 0.1.0
+version: 0.1.0-1c62490
```

Comment:
probably bump this to 0.2.0

Reply:
Yes, I was thinking about releasing all the services but maybe once they get merged to master.

Reply:
no, I mean the working version of the chart should be `>0.1.0`, i.e. `0.2.0-1234` is a development version of `0.2.0`.

Reply:
Yes, that's true -> updated.